.github/workflows/providers-core.yml

```diff
@@ -7,13 +7,20 @@ on:
   schedule:
     # run once per day
     - cron:  '19 7 * * *'
-concurrency: 
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'branch or git ref to use for the build'
+        required: true
+        default: 'test/providers'
+
+concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 jobs:
   provider-test:
-    name: Provider Test
-    if: ${{ github.repository == 'kopia/kopia' }}
+    name: Core Providers Test
+    if: ${{ github.repository == 'kopia/kopia' && !github.event.pull_request.head.repo.fork }}
```

Comment:
How about ?:

```
if: ${{ github.event_name == 'schedule' }}
```

Reply:
This needs to run on various events, including branch and tag pushes, not just PRs.